Utils/azuretools-core/src/com/microsoft/azuretools/utils/WebAppUtils.java

```diff
@@ -371,60 +371,34 @@ public ResourceGroup getRg() {
+     * Docker web apps are not included.
      */
-    public static String getJDKVersion(@NotNull final WebApp webApp) {
-        switch(webApp.operatingSystem()) {
-            case WINDOWS:
-                return webApp.javaVersion().toString();
-            case LINUX:
-                final String linuxVersion = webApp.linuxFxVersion();
-                if (linuxVersion == null) {
-                    return DEFAULT_VALUE_WHEN_VERSION_INVALID;
-                }
-
-                final String[] versions = linuxVersion.split("-");
-                return versions.length != 2 ? linuxVersion : versions[1];
-            default:
-                return DEFAULT_VALUE_WHEN_VERSION_INVALID;
-        }
+    public static boolean isJavaWebApp(@NotNull WebApp webApp) {
+        return webApp.javaVersion() != JavaVersion.OFF || StringUtils.containsIgnoreCase(webApp.linuxFxVersion(), "jre8");
     }
 
     /**
-     * app.linuxFxVersion() is the only API we could get the version info of a Linux web app.
-     * It returns "Tomcat|8.5-jre8" if it is a Linux web app with the web container Tomcat.
-     * Tomcat is the only supported web container.
-     * If the web app is a Java SE web app, which has no web container, linuxFxVersion() returns "Java|8-jre8".
-     * And we will return N/A for those kind of web apps.
+     * For a Windows web app, APIs are separated to get jdk information and web container information.
+     * For a Linux web app, API app.linuxFxVersion() returns a combined information, like
+     * "Tomcat|8.5-jre8" if it is a Linux web app with the web container Tomcat.
+     * We return a combined and refactored information as Java Runtime.
      */
-    public static String getWebContainer(@NotNull final WebApp webApp) {
-        switch(webApp.operatingSystem()) {
+    public static String getJavaRuntime(@NotNull final WebApp webApp) {
+        final String delimiter = "  ";
+        switch (webApp.operatingSystem()) {
             case WINDOWS:
-                return String.join(" ", webApp.javaContainer(), webApp.javaContainerVersion());
+                final String refactoredJavaVersion = "Java" + webApp.javaVersion().toString();
+                return String.join(delimiter, webApp.javaContainer(), webApp.javaContainerVersion(), refactoredJavaVersion);
             case LINUX:
                 final String linuxVersion = webApp.linuxFxVersion();
-                final String[] versions = linuxVersion.split("-");
-                if (versions.length != 2) {
-                    return linuxVersion;
-                }
-                if (StringUtils.containsIgnoreCase(versions[0], "tomcat")) {
-                    return versions[0].replace("|", " ");
+                if (linuxVersion == null) {
+                    return DEFAULT_VALUE_WHEN_VERSION_INVALID;
                 } else {
-                    return "N/A";
+                    return linuxVersion.replace("|", delimiter).replace("-", delimiter);
```

Comment:
Why bother to format the value? Is it better to show what SDK returns?

Reply:
Are you suggesting that we use the format to combine the Windows Runtime? Like `Tomcat|8.5-Jre8`, I don't think it is a good format to show in our UI to users, especially the `|` between web container and its version. Maybe some suggestions about the formats?

Reply:
I prefer not to format the value returned by SDK. For me, it's natural to combine two values returned by SDK in some format, and I'd rather not to do some tricky things inside each of them. 
@jdneo Do you have some suggestion?

Reply:
Or can we leverage the format in Azure portal for consistence instead of creating a new format by our own? For Linux Web App, I see the format is "Tomcat 8.5 (JRE 8)".

Reply:
And "JAVA 8 jre8" also looks wired. 🤦‍♂️

Reply:
Agree to use the format `Tomcat 8.5 (JRE8)` for Linux web apps and `Tomcat 8.5 (Java1.8)` for Windows web apps.
For Java SE web app, we will just show `JRE8`.